### PR TITLE
wip

### PR DESCRIPTION
**Pull request by JeanMertz**
🚧

### CODE REVIEW

File: features/webmachine/collection/get.feature [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+  Background:
+    Given the route "/items" points to the "items" resource
+
+  Scenario: Accept header "application/json" returns "200 OK" and a body of type "application/json"
+    Given the following "items" resource:
+      """ruby
+      class ItemsResource < Webmachine::Resource
+        include Eipiai::Resource
+
+        def object
```
Comment (jurriaan) [inline features/webmachine/collection/get.feature]: Maybe a bit to common of a name maybe?

Comment (JeanMertz) [inline features/webmachine/collection/get.feature]: Hmm. Suggestions?

File: features/step_definitions/webmachine_steps.rb [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+  Eipiai::Test::App.add_route ['api'], Eipiai::ApiResource
+  Eipiai::Test::App.add_route ['health'], Eipiai::HealthResource
+end
+
+After do
+  # Remove any existing resource objects
+  #
+  Eipiai::Test.class_exec do
+    constants.each do |constant|
+      remove_const(constant) if constant =~ /\S+Resource/
```
Comment (jurriaan) [inline features/step_definitions/webmachine_steps.rb]: Constants never include spaces, right? Just use `ends_with?`

File: lib/eipiai/roar/ext/hal.rb [hunk 3]
```diff
@@ -50,6 +50,8 @@ def to_hash(*)
               hash['_links']['curies'] = curies.to_a if curies.any?
             end
           end
+
+          alias to_h to_hash
```
Comment (jurriaan) [inline lib/eipiai/roar/ext/hal.rb]: alias_method?

Comment (JeanMertz) [inline lib/eipiai/roar/ext/hal.rb]: Rubocop says: Nope.

File: features/step_definitions/webmachine_steps.rb (hunk 2 above)
Comment (JeanMertz) [inline features/step_definitions/webmachine_steps.rb]: 👍

**Pull request closed without merging**